src/gui/qgsidentifymenu.cpp

```diff
@@ -282,7 +283,12 @@ void QgsIdentifyMenu::addVectorLayer( QgsVectorLayer *layer, const QList<QgsMapT
   if ( !createMenu )
   {
     // case 1
-    QString featureTitle = results[0].mFeature.attribute( layer->displayField() ).toString();
+
+    QgsExpressionContext context( QgsExpressionContextUtils::globalProjectLayerScopes( layer ) );
```

Comment:
Can't we reuse this context throughout ::addVectorLayer, instead of the code duplication?

Reply:
Yes. Thanks